keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -1764,7 +1753,35 @@ notices this, the current scheduling attempt for the pod must stop and the pod
 needs to be put back into the work queue. It then gets retried whenever a
 ResourceClaim gets added or modified.
 
-The following extension points are implemented in the new claim plugin:
+The following extension points are implemented in the new claim plugin. Some of
+them invoke API calls to create or update objects. This is done to simplify
+error handling: a failure during such a call puts the pod into the backoff
+queue where it will be retried after a timeout. The downside is that the
```

Comment:
This is troubling. I think it needs more thought. I think not disrupting scheduling of other pods needs to be a graduation criteria.

Reply:
It's already almost solved: https://github.com/kubernetes/kubernetes/pull/121876

A discussion around PostFilter is on-going on Slack.

Normally, architectural changes would be first described in a KEP and then get implemented, but changes to existing code are easier to explore, show and comment on via a PR. Let's update this text once the code discussions are done.

/hold

> I think not disrupting scheduling of other pods needs to be a graduation criteria.

Agreed.

Reply:
Added and text updated.

We decided to not invest undue effort (and introduce complexity) for addressing the PostFilter deallocate edge case.

Reply:
/hold cancel